frontend/public/i18n/de.json

```
@@ -1,10 +1,36 @@
 {
   "PCTS_LOGO_ALT": "Puzzle PCTS Logo",
   "LOGOUT": "Abmelden",
+
+  "GENERAL": {
+    "ADD": "Hinzufügen",
+    "EDIT": "Ändern",
+    "CANCEL": "Abbrechen",
+    "SAVE": "Speichern"
+  },
+
+  "VALIDATION": {
+    "REQUIRED": "Muss ausgefühlt sein",
+    "DATE_IS_IN_FUTURE": "Datum muss in der Vergangenheit liegen",
+    "INVALID_ENTRY": "Ungültige Eingabe",
+    "MATDATEPICKERPARSE": "Ungültiges Datum",
+    "NOT_AVAILABLE": "\"{{value}}\" nicht verfügbar"
+  },
+
   "MEMBER": {
     "NAME": "Name",
-    "BIRTHDATE": " Geburtsdatum",
+    "FIRSTNAME": "Vorname",
```

```suggestion
    "FIRST_NAME": "Vorname",
```
Just to keep it consistent with `LAST_NAME`.